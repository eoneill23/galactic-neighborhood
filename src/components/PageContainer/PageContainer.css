.planet-container {
  background-color: black;
  display: flex;
  height: 600px;
  flex-direction: row;
}

.planet-fact {
  background: gray;
  color: white;
  height: 175px;
  margin-top: 100px;
  padding: 10px;
  width: 60%;
}

.planet-img {
  height: 500px;
  width: 500px;
}