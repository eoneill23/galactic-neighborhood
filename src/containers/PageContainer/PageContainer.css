.planet-container {
  background-color: black;
  display: flex;
  height: 600px;
}

.planet-fact {
  color: white;
  padding: 10px;
  width: 60%;
}

.planet-img {
  height: 500px;
  width: 500px;
}