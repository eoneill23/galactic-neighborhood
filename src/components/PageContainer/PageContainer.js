import React from 'react';
import './PageContainer.css';

const PageContainer = ({ fact, imgUrl, factUrl, factSite }) => {

  return (
    <section className='planet-container'>
        <img src={imgUrl} className='planet-img'/>
        <div className='planet-fact'>
        {fact}
          <p>Fact courtesy of <a href={factUrl} target='_blank'>{factSite}</a>.</p>
        </div>
    </section>
  )
}

export default PageContainer;