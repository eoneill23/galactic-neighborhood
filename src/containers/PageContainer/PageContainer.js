import React from 'react';
import './PageContainer.css';

const PageContainer = ({ fact, imgUrl, factUrl, factSite }) => {


  return (
    <section>
      <img src={imgUrl} className='planet-img'/>
      <p className='planet-fact'>{fact}</p>
      <p>Fact courtesy of <a href={factUrl}>{factSite}</a></p>
    </section>
  )
}

export default PageContainer;