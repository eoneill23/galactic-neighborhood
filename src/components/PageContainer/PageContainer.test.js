import React from 'react';
import { shallow } from 'enzyme';
import { PageContainer } from './PageContainer';

describe('PageContainer', () => {

  it('should match the snapshot with all data passed in correctly', () => {
    let mockCalculateWeight = jest.fn();
    let wrapper = shallow(
    <PageContainer 
      fact={'Closest to the sun.'}
      imgUrl={'google.com'}
      factUrl={'space.com'}
      factSite={'space.com'}
      multiplier={2}
      calculateWeight={mockCalculateWeight}
    />);

    expect(wrapper).toMatchSnapshot();
  });
});