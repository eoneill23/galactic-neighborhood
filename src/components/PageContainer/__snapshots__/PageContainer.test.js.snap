// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContainer should match the snapshot with all data passed in correctly 1`] = `
<section
  className="planet-container"
>
  <img
    alt="Planet"
    className="planet-img"
    src="google.com"
  />
  <div
    className="planet-fact"
  >
    Closest to the sun.
    <p>
      Fact courtesy of 
      <a
        href="space.com"
        target="_blank"
      >
        space.com
      </a>
      .
    </p>
    <Connect(WeightForm)
      calculateWeight={[MockFunction]}
      multiplier={2}
    />
  </div>
</section>
`;
